### PR TITLE
MAID-3119: docs/license: update LICENSE details

### PR DESCRIPTION
**Pull request by pravarseth**
Made license text more consistent with other repos

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,8 +12,7 @@
 
 ## License
 
-This is dual-licensed under the Modified BSD ([LICENSE-BSD](LICENSE-BSD) https://opensource.org/licenses/BSD-3-Clause) or the MIT license ([LICENSE-MIT](LICENSE-MIT) https://opensource.org/licenses/MIT) at your option.
-
+This SAFE Network library is dual-licensed under the Modified BSD ([LICENSE-BSD](LICENSE-BSD) https://opensource.org/licenses/BSD-3-Clause) or the MIT license ([LICENSE-MIT](LICENSE-MIT) https://opensource.org/licenses/MIT) at your option.
```
Comment (bochaco) [inline README.md]: @pravarseth , shall we consider a small change here as this is not really a library but a SAFE Network application?

Comment (nadiaburborough) [reply README.md]: @bochaco so are you suggesting we change it to:
"This SAFE Network `application` is dual-licensed ..."

Comment (bochaco) [reply README.md]: yes @nadiaburborough , so it's more accurate

**Comment by bochaco** [comment]
I'll be making this change in a separate PR as this one has be stalled for long now.

**Pull request closed without merging**